### ISSUE TITLE
WebView requests blocked by CORS policy 

### ISSUE THREAD

**Issue by GuillaumeDesforges** [microsoft/vscode-remote-release#3076]
```
Version: 1.46.0-insider (user setup)
Commit: 9e439b13d5784538333bee044ca1d1a992bc1c90
Date: 2020-05-28T18:44:47.151Z
Electron: 7.3.0
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.19631
```

- VSCode Version: 1.46.0-insider
- Local OS Version: Windows 10.0.19631
- Remote OS Version: NixOS 20.03
- Remote Extension/Connection Type: SSH

Using https://github.com/microsoft/vscode-java-test/, see issue https://github.com/microsoft/vscode-java-test/issues/984

Steps to Reproduce:

1. Connect to remote
2. Install VSCode Java Test Runner extension
3. Create Java Project with test
4. Run test
5. Open Java Test Report panel
6. Click on the "expend" icon -> it does not work

I can see the Cross-Origin issue in devtools:

![image](https://user-images.githubusercontent.com/1882000/83238646-0be65700-a197-11ea-9c09-a702d3a32acc.png)

Could not answer those questions as I have no dev environment locally, but I suppose that it would work since it would read the files directly locally.

Does this issue occur when you try this locally?: Yes/No
Does this issue occur when you try this locally and all extensions are disabled?: Yes/No

**Comment by GuillaumeDesforges** [comment]
https://github.com/microsoft/vscode/issues/72900 seems to be of interest to me

**Comment by Chuxel** [comment]
//cc: @mjbvz

**Comment by mjbvz** [comment]
Seems like duplicate of https://github.com/microsoft/vscode/issues/99190

Are you still seeing this in the latest VS Code insiders builds

**Comment by GuillaumeDesforges** [comment]
Yup, it works
